requirements/deployment.in

```diff
@@ -22,6 +24,7 @@ elasticsearch<7.11  # Keep pinned to the deployed ES version
 govdelivery==1.4.0
 Jinja2==3.1.2
 lxml==4.9.1
+newrelic==8.5.0
```

Comment:
Merging this into the main list of requirements necessitates rewriting [this part of the docs](https://cfpb.github.io/consumerfinance.gov/debugging-monitoring/#monitoring-using-new-relic).